src/main/java/com/minecolonies/network/messages/OpenInventoryMessage.java

```diff
@@ -96,15 +156,27 @@ public IMessage onMessage(OpenInventoryMessage message, MessageContext ctx)
             case INVENTORY_CITIZEN:
                 InventoryCitizen citizenInventory = ((EntityCitizen) player.worldObj.getEntityByID(message.entityID)).getInventoryCitizen();
                 if(!StringUtils.isNullOrEmpty(message.name))
-                    citizenInventory.setCustomName(message.name);   //SetInventoryName
+                {
+                    citizenInventory.setCustomName(message.name);
+                }
                 player.displayGUIChest(citizenInventory);
                 break;
             case INVENTORY_CHEST:
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this switch case number of lines from 7 to at most 5, for example by extracting code into methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1151)